src/vmm/src/linux/vstate.rs

```diff
@@ -97,6 +102,11 @@ pub enum Error {
     #[cfg(target_arch = "x86_64")]
     /// Error configuring the general purpose registers
     REGSConfiguration(arch::x86_64::regs::Error),
+    #[cfg(target_arch = "aarch64")]
+    /// Error setting up the global interrupt controller.
+    SetupGIC(arch::aarch64::gic::Error),
```

Comment:
I must've accidentally included this as a result of a rebase. Will remove from this commit.